server.ts

```diff
@@ -209,6 +211,26 @@ io.on("connection", (socket) => {
 			timerRequest({ roomName, timerStore, socket });
 		}
 	});
+
+	// handler breakTimer : on emit of "breakTimer" from the cf-frontend
+	socket.on("breakTimer", (breakTimer: EmitBreakTimerArgs) => {
+		console.log("-------------------");
+		console.log("Console log from the 'breakTimer' emit event");
+		console.log({
+			Username: `Client's user name is ${breakTimer.userName}`,
+			roomName: `Client's roomName is ${breakTimer.roomName}`,
+		});
+	});
```

Comment:
```suggestion
		console.log("Console log from the 'breakTimer' emit event", {
			Username: `Client's user name is ${breakTimer.userName}`,
			roomName: `Client's roomName is ${breakTimer.roomName}`,
		});
	});
```

If we are logging this, maybe we combine the console logs?